service/history/timerQueueProcessorBase.go

```diff
@@ -590,26 +590,3 @@ func (t *timerQueueProcessorBase) deleteWorkflowVisibility(
 	}
 	return backoff.Retry(op, persistenceOperationRetryPolicy, common.IsPersistenceTransientError)
 }
-
-func (t *timerQueueProcessorBase) getTimerTaskType(
```

Comment:
I think this function is useful for logging & debugging purpose, can we keep it?

Reply:
sure. if so I will keep it